pkg/services/policer/process.go

```diff
@@ -48,15 +48,25 @@ func (p *Policer) shardPolicyWorker(ctx context.Context) {
 				return
 			default:
 				addr := addrs[i]
+				if p.objsInWork.inWork(addr) {
+					// do not process an object
+					// that is in work
+					continue
+				}
+
 				addrStr := addr.EncodeToString()
```

Comment:
Not related to the PR theme, but can we avoid encoding and use `oid.Address` as cache keys?

Reply:
hm, why not, changed to store the same type in cache and map